src/search/search_node_info.cc

```diff
@@ -1,10 +1,6 @@
 #include "search_node_info.h"
 
-static const int pointer_bytes = sizeof(void *);
-static const int info_bytes = 3 * sizeof(int) + sizeof(StateID);
-static const int padding_bytes = info_bytes % pointer_bytes;
-
 static_assert(
-    sizeof(SearchNodeInfo) == info_bytes + padding_bytes,
-    "The size of SearchNodeInfo is larger than expected. This probably means "
-    "that packing two fields into one integer using bitfields is not supported.");
+    sizeof(SearchNodeInfo) == 4 + sizeof(StateID) + sizeof(OperatorID),
```

Comment:
This ignores word alignment now.

Reply:
I thought so as well, but this version of the code compiles. My guess is that there will be padding once more than one SearchNodeInfo is concatenated in memory.